hack/verify-licenses.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `hack/verify-licenses.sh`.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+
+
+# Ensure that we find the binaries we build before anything else.
+export GOBIN="${KUBE_OUTPUT_BINPATH}"
+PATH="${GOBIN}:${PATH}"
+
+
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
+
+
+# Color Codes
+RED="\033[0;31m"
+GREEN="\033[0;32m"
+YELLOW="\033[0;33m"
+CAYAN="\033[0;36m"
+GRAY="\033[0;37m"
+NC="\033[0m" # No Color
+
+
+declare -a packages_flagged
+declare -a allowed_licenses
+
+LICENSES_JSON="/tmp/$(cd /tmp && mktemp licenses_json.XXXXXX)"
+LICENSES_CSV="/tmp/$(cd /tmp && mktemp licenses_csv.XXXXXX)"
+LICENSES_DUMP="/tmp/$(cd /tmp && mktemp licenses_dump.XXXXXX)"
+
+# Install go-licenses
+printf '~%.0s' {1..120}; echo
+echo -e "\nInstalling go-licenses..."
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
+  go install github.com/google/go-licenses@v1.0.0
```

Comment:
looks like https://github.com/kubernetes/git-sync/blob/8dea3f4be563ff40656caf337574c4bb9eac30d3/Makefile#L126 we pull from HEAD, let's do that please

Reply:
```suggestion
  git clone https://github.com/google/go-licenses.git
  cd go-licenses
  go build -o $GOPATH/bin
```

^ this is working when I tried locally. Is `$GOPATH` set in the CI? 🤔 If that's the case then it should work there too.

Reply:
@dims 

> looks like https://github.com/kubernetes/git-sync/blob/8dea3f4be563ff40656caf337574c4bb9eac30d3/Makefile#L126 we pull from HEAD, let's do that please

For the step mentioned in the link ^, git-sync project is creating an empty [tools.go](https://github.com/kubernetes/git-sync/blob/release-3.x/tools.go) file which is just importing the go-licenses module.

Since, the error was relate git remote origin url & not go package install. I kept the go install as it is.

I'll re-iterate if changes are still required.

(Thanks @RinkiyaKeDad too for the input 🙂 )